activity_calendar/tests/test_models.py

```diff
@@ -174,6 +174,14 @@ def test_time_shift_in_get_occurences(self):
         pass
 
 
+class ActivityTestCase(TestCase):
+
+    def test_get_all_activity_moments(self):
```

Comment:
This is left empty; is that intentional?

Reply:
I'm currently writing that. Like I said in the chat. I'm still writing some additional test cases